### PR TITLE
Change variable name

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## Related Issues and Dependencies

Related-To: https://github.com/thoth-station/workflow-helpers/pull/67

## This introduces a breaking change

- [ ] Yes
- [x] No

## Test or Stage Environment

- [x] Pull Requests added content to TEST/STAGE environment

## This Pull Request implements

Modify variable name

### CODE REVIEW

File: adviser/base/argo-workflows/trigger-integration.yaml [hunk 1]
```diff
@@ -14,9 +14,9 @@ spec:
           - name: "THOTH_ADVISER_METADATA"
       outputs:
         parameters:
-          - name: origin
+          - name: metadata_origin
```
Comment (harshad16) [inline adviser/base/argo-workflows/trigger-integration.yaml:17]: does adding a default value fix the issue?

Comment (pacospace) [reply adviser/base/argo-workflows/trigger-integration.yaml:17]: https://github.com/thoth-station/workflow-helpers/pull/67#issuecomment-691073150

Comment (harshad16) [inline adviser/base/argo-workflows/trigger-integration.yaml:17]: if we are changing this value, we should change this as well  https://github.com/thoth-station/thoth-application/blob/c45e9f6395d576de8e0e346f29c1a66959a7e968/adviser/base/openshift-templates/adviser.yaml#L281
right ?

Comment (pacospace) [reply adviser/base/argo-workflows/trigger-integration.yaml:17]: nice catch! thanks!!

**Review by harshad16 — APPROVED** [review]
lgtm
thanks for the fix :100: :rocket:

**Pull request merged**